tests/scripts/thread-cert/border_router/test_multi_ail.py

```diff
@@ -124,6 +124,25 @@ def test_multi_backbone_infra(self):
         self.assertTrue(br2.ping(br3_onlink_ula, backbone=True))
         self.assertTrue(br3.ping(br2_onlink_ula, backbone=True))
 
+    def test_adjcent_peer_br_count_less_than_netdata_peer_br_count(self):
+        """This test ensures that the adjcent_peer_br count is less than netdata_peer_br count in Mutil-AIL topology."""
+        br1: OtbrNode = self.nodes[self.BR1]
+        br2: OtbrNode = self.nodes[self.BR2]
+        br3: OtbrNode = self.nodes[self.BR3]
+
+        for br in [br1, br2, br3]:
+            br.start()
+        self.simulator.go(config.BORDER_ROUTER_STARTUP_DELAY)
+
+        self.assertEqual(len(br1.get_br_routers_peer_ip_addresses()), 0)
+        self.assertEqual(len(br1.get_br_peers_rloc16s()), 2)
+
+        self.assertEqual(len(br2.get_br_routers_peer_ip_addresses()), 1)
+        self.assertEqual(len(br2.get_br_peers_rloc16s()), 2)
+
+        self.assertEqual(len(br3.get_br_routers_peer_ip_addresses()), 1)
+        self.assertEqual(len(br3.get_br_peers_rloc16s()), 2)
```

Comment:
I think it's also worth verifying the output of dbus because that's where the telemetry is collected.

Reply:
Thanks for the suggestions!
There's a plan to test dbus related in ot-br-posix repo, I will focus on the BR count test here.

Reply:
> There's a plan to test dbus related in ot-br-posix repo.

AFAIK there isn't an easy way to set up multi AIL environment in ot-br-posix. Could you share your idea?

Reply:
initial idea is to use a shell script to create docker network and run docker containers to simulate OTBR nodes